migrations/20200201140355_initial.up.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE uris (
+  uri text PRIMARY KEY
+);
+
+CREATE TABLE connections (
+  recipient_uri text PRIMARY KEY
+    REFERENCES uris (uri)
+    ON DELETE CASCADE
+    ON UPDATE RESTRICT,
```

Comment:
Wouldn't connections / uris disallow updates even when not referenced? If yes wouldn't it be more appropriate to provide immutability via `REVOKE` / triggers, rather than through foreign keys? Or are there other reasons for `ON UPDATE RESTRICT` constraints?

Reply:
I just don't like triggers tbh :)
It just to show via the schema that the connection cannot be updated.
Updates won't be allowed on API level

Reply:
I will merge, but we can keep changing this initial migration (rather than creating a new one), to see the final schema for v0.0.1 protocol in one place, until it's released on some live server.